components/form/builder/src/index.js

```diff
@@ -18,21 +18,22 @@ import AtomSpinner, {AtomSpinnerTypes} from '@s-ui/react-atom-spinner'
 import {getUpdatedFormState} from './mapper/formState'
 
 const FormBuilder = ({
+  applyEmptyFields,
```

Comment:
what is the new prop about? by its name, I'm not sure it's self-explanatory. Also, should it default to `false`?

Reply:
This prop apply form builder logic like apply rules and set initialFields values for empty fields, for example, if has defined `{categoryId: '1', subCategoryId: ''}` without `applyEmptyFields` prop, will only apply rules and form logic for categoryId. The name probably could be improved, any suggestion 😄?

Reply:
By default is falsy to not break actual behaviour

Reply:
my 2 cents if I may 😉

`shouldAlwaysApplyRules`
`forceRulesOnAllFields`

I miss the `rules` part on `applyEmptyFields` and I think it's key to understand the purpose of the prop...

Reply:
I like `forceRulesOnAllFields`, I think it's clearer, thank you!